src/commands/make-publish.ts

```diff
@@ -1,20 +1,28 @@
 import { grey, cyan } from 'chalk';
-import { Command } from '@oclif/command';
+import { Command, flags } from '@oclif/command';
 
 import { loadConfig } from '../config';
+import { CONFIG_FILENAME } from '../constants';
 import { printInfo, printLine } from '../util/io';
 import * as fileMakerService from '../service/fileMaker';
 
 class MakePublish extends Command {
   static description = 'Publish migration templates files.';
-
+  static flags = {
+    config: flags.string({
+      char: 'c',
+      default: CONFIG_FILENAME,
```

Comment:
Shouldn't have any defaults. We should not run the command if `-c` is run without providing the configuration file name.